src/containers/my-quizzes/edit-quiz-container/EditQuizContainer.tsx

```diff
@@ -30,8 +33,16 @@ const EditQuizContainer = ({ questions, setQuestions }: QuizContentProps) => {
   const { openModal } = useModalContext()
   const [title, setTitle] = useState<string>('')
   const [description, setDescription] = useState<string>('')
+  const [category, setCategory] = useState<string>('')
   const [isCreationOpen, setIsCreationOpen] = useState<boolean>(false)
 
+  const onCategoryChange = (
+    _: React.SyntheticEvent,
+    value: CategoryNameInterface | null
+  ) => {
+    setCategory(value?._id ?? '')
+  }
+
```

Comment:
Try to increase test coverage, at least on your new code
(You can see all coverage on Sonar, or locally at vitest report)

Reply:
fixed